src/sentry/sentry_metrics/models/metricsextractionrules.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import annotations
+
+from typing import Any
+
+from django.db import models
+from django.db.models import CASCADE
+from django.utils import timezone
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import ArrayField, FlexibleForeignKey, Model, region_silo_model
+from sentry.db.models.fields.hybrid_cloud_foreign_key import HybridCloudForeignKey
+from sentry.models.project import Project
+from sentry.sentry_metrics.configuration import HARD_CODED_UNITS
+from sentry.sentry_metrics.extraction_rules import MetricsExtractionRule
+
+
+@region_silo_model
+class SpanAttributeExtractionRuleCondition(Model):
+    __relocation_scope__ = RelocationScope.Organization
+
+    date_modified = models.DateTimeField(default=timezone.now)
+    date_added = models.DateTimeField(default=timezone.now)
```

Comment:
You probably want to inherit from `DefaultFieldsModel`. By default django won't manage `date_modified` fields. However, `DefaultFieldsModel` will manage it's `date_updated` field.